module/power_domain/src/mod_power_domain.c

```diff
@@ -294,7 +294,7 @@ static const unsigned int mod_pd_cs_level_state_shift[MOD_PD_LEVEL_COUNT] = {
  * Internal variables
  */
 static struct mod_pd_ctx mod_pd_ctx;
-static const char driver_error_msg[] = "[PD] Driver error %e in %s @%d\n";
+static const char driver_error_msg[] = "[PD] Driver error %d/%s in %s @%d\n";
```

Comment:
I think `%s (%d)` is likely to prove a bit more readable.

Reply:
i'll fix all occurrences.